### PR TITLE
switching to tactile receiver 

### PR DESCRIPTION
**Pull request by ugocupcic**
gets rid of the deprecated shadowhand ros lib

### CODE REVIEW

File: sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py [hunk 1]
```diff
@@ -29,13 +29,14 @@ class SrHandCommander(SrRobotCommander):
 
     __set_force_srv = {}
 
-    def __init__(self, name="right_hand"):
+    def __init__(self, name="right_hand", prefix="rh"):
```
Comment (ugocupcic) [inline sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py:32]: bit of a duplicated information here but I wasn't sure how to get rid of it. We need to look carefully at our launch files and harmonize everything.

**Comment by ugocupcic** [comment]
@kirstyellis I hope you didn't start on this on Friday. I couldn't see any branch so thought you hadn't.

File: sr_hand/scripts/sr_hand/tactile_receiver.py [hunk 2]
```diff
@@ -26,16 +26,24 @@
...
+        """
+        Receives the tactile information from a Shadow Hand
+        @param prefix - prefix for the tactile topic
+        """
         self.tactile_type = self.find_tactile_type()
         self.tactile_state = None
-        
+
+        #appends trailing slash if necessary
+        if prefix[-1] is not "/":
```
Comment (toliver) [inline sr_hand/scripts/sr_hand/tactile_receiver.py:38]: doesn't this throw exception with no prefix?

Comment (AndriyPt) [inline sr_hand/scripts/sr_hand/tactile_receiver.py:38]: @toliver  yes, you are right it will throw 
IndexError: string index out of range

Comment (ugocupcic) [inline sr_hand/scripts/sr_hand/tactile_receiver.py:38]: #243

**Pull request merged**